RaccoonPG.py

```
@@ -0,0 +1,389 @@
+
+    #TO BE USED ON BOT RESTART#
+    def passive_xp(self, num_msg):
+        self.EXP += float(num_msg * (1/10000))
+
+    #TO BE USED WHEN ACTIVE#
+    def passive_xp_type(self):
+        self.EXP += float(1/10000)
+
+class Swordsman(Raccoon):
+
+    def set_slow_xp(self, EXPcap, LVL):
```

Ruff (0.14.0)

Unused method argument: `EXPcap`

(ARG002)